### ISSUE TITLE
Create a new team interface not intuitive

### ISSUE THREAD

**Issue by thetechnologyvault** [nanobox-io/dashboard-ui-plan-funnel#1]
The interface for creating a new team should be more intuitive.

When I was creating a new team

Suggested recommendations:
- Make a clearer separation between My Teams and Create a the Create a New Team section
- Make "Create a New Team" the header for the section
- Remove the "Create a new team" prompt
- Move the **Notice: After you create...** below the "Team Name" input field
- Make the **Cancel** and **Save Changes** buttons visible on page load; gray out **Save Changes** button until a name is entered.

**Comment by tylerflint** [comment]
Agreed. there was an interim workflow for this that was more intuitive, but sometime in the last week it was pulled into a different flow without Marks direction. We need to either revert to the simpler workflow or have Mark address this process. 

@tolmark12 thoughts?

**Comment by jedgalbraith** [comment]
Also, just to be clear. This feedback is related to the team create in the user settings, not the funnel.
![screen shot 2017-06-03 at 5 43 08 pm](https://cloud.githubusercontent.com/assets/313947/26757737/4199f9f2-4884-11e7-8952-dce9dbb8ab5b.png)

**Comment by Tolmark12** [comment]
@jedgalbraith was good enough to implement this without a design, we'll come back and add a design that takes these things into consideration
